### DISCUSSION TITLE
Bentley sample model

### DISCUSSION THREAD

**Discussion by ShivaGugulothu** [iTwin/community#35] [Q&A]
Can it possible to download the bentley supplied sample imodels

**Comment by aruniverse** [comment]
Which sample imodels are you looking to download and use?

**Comment by ShivaGugulothu** [comment]
The models which bentley showing as bentley supplied sample imodels. In bentley they are Retail building sample, Bay town process plant, House sample etc

**Comment by ShivaGugulothu** [comment]
How can i download those models?

**Reply by Johannes-repos** [reply]
have look here:
https://github.com/orgs/iTwin/discussions/48
